content/docs/contributing/release-process.md

```diff
@@ -578,3 +578,32 @@ Other than the different `cert-manager/release` tag and `cmrel` version, you can
 is used for 1.6 and 1.7 - just remember to change the version of `cmrel` you install!
 
 [older-release-process]: https://github.com/cert-manager/website/blob/6fa0db74de0ae17d7be638a08155d1b4e036aaa9/content/en/docs/contributing/release-process.md?plain=1
+
+## Documentation Process
+
+Everytime a PR is opened on the cert-manager/website repository, you will need
+to check that the "base branch" is correct.
+
+|                Description                 |  Base branch   |     Folder     |
+|--------------------------------------------|----------------|----------------|
+| Documentation PR for an existing feature   | `master`       | `content/docs` |
+| Documentation PR for an unreleased feature | `release-next` | `content/docs` |
+
+When creating a documentation PR for an unreleased feature, you will need to
+fast-forward the `release-next` branch to `master` before asking for a review of
+the PR:
```

Comment:
Copied from https://github.com/cert-manager/website/pull/1081#discussion_r1114395964:

> I do not agree with this.
> It may be desirable to bring release-next up to date before submitting your release-next PR,
> but usually only if the new feature you are documenting might conflict with some changes which have meanwhile been merged into the live documentation.
> 
> In any case, release-next doesn't have to be "fast-forwarded"...just merged....unless there's a way to create a PR which really does fast forward the commits from one branch to another. Is that possible?
> 
> This release process documentation should be in a separate PR.

Reply:
I agree. I think I misunderstood the implications of not having `release-next` up-to-date. Even in my case, I didn't have to update `release-next` to match `master`.

- I will rephrase to make it an optional step only needed if the changes you intend to make to the documentation relies on changes that were made recently to master and aren't in release-next.
- I will use the word "merge master into release-next" instead of "fast-forward release-next to master". I agree with you, it can't be fast-forwarding and merge commits are needed.